packages/dd-trace/src/agentless/exporter.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const log = require('../log')
+
+class LogExporter {
+  send (queue) {
+    log.JSON({ traces: queue })
```

Comment:
This should not use `log` since it's configured by the user and the actual format cannot be controlled. I would say to write directly to `process.stdout`.